README.md

```diff
@@ -89,6 +89,46 @@ vg.fillColor(nvg.rgba(255,192,0,255));
 vg.fill();
 ```
 
+## Building a Program with nanovg-zig
+
+Here's how to integrate `nanovg-zig` into your Zig project:
+
+1.  **Add the library as a dependency:**
+
+    Run the following command in your project directory to fetch and save `nanovg-zig` as a dependency:
+
+    ```bash
+    zig fetch --save git+https://github.com/fabioarnold/nanovg-zig.git
+    ```
+
+2.  **Import the library in your `build.zig` file:**
+
+    Add the following code snippet to your `build.zig` file. This creates a dependency on `nanovg-zig` and imports the `nanovg` module into your executable:
+
+    ```zig
+    const nanovg_zig = b.dependency("nanovg-zig", .{
+        .target = target,
+        .optimize = optimize,
+    });
+    exe.root_module.addImport("nanovg", nanovg_zig.module("nanovg"));
+    ```
+
+    Replace `exe` with the name of your ExecutableOptions instance.
```

Comment:
Hm, `exe` should be of type `*Step.Compile`, no? `addExecutable` takes `ExecutableOptions` as argument.

Maybe we could say:
> Replace `exe` with the name of your target executable.